README.md

```diff
@@ -59,36 +59,45 @@ We are breaking down the various parts that we need to migrate to Swift in the s
-|                     | `onRefresh`                                         |        |
-|                     | `onUpdate`                                          |        |
-|                     | `refreshInternal`                                   |        |
-|                     | `stickyFeatures`                                    |        |
-| Instance methods    | `setDatafile`                                       |        |
-|                     | `setStickyFeatures`                                 |        |
-|                     | `getRevision`                                       |        |
-|                     | `getFeature`                                        |        |
-|                     | `getBucketBy`                                       |        |
-|                     | `getBucketValue`                                    |        |
-|                     | `isReady`                                           |        |
+|                     | `onReady`                                           | ✅     |
+|                     | `onRefresh`                                         | ✅     |
+|                     | `onUpdate`                                          | ✅     |
+|                     | `refreshInternal`                                   | ✅     |
+|                     | `stickyFeatures`                                    | ✅     |
+|                     |                                                     |        |
+| Instance methods    | `constructor` missing fetch datafile content        | ⚠️      |
+|                     | `setDatafile` removed to workaround init issues     | ✅     |
+|                     | `setStickyFeatures`                                 | ✅     |
+|                     | `getRevision`                                       | ✅     |
+|                     | `getFeature`                                        | ✅     |
+|                     | `getBucketKey`                                      | ✅     |
+|                     | `getBucketValue`                                    | ✅     |
+|                     | `isReady`                                           | ✅     |
 |                     | `refresh`                                           |        |
 |                     | `startRefreshing`                                   |        |
 |                     | `stopRefreshing`                                    |        |
-|                     | `evaluateFlag`                                      |        |
-|                     | `isEnabled`                                         |        |
 |                     | `evaluateVariation`                                 |        |
-|                     | `getVariation`                                      |        |
+|                     | `getVariation`                                      | ✅     |
+|                     | `getVariationBoolean`                               |        |
```

Comment:
a recent refactoring in https://github.com/fahad19/featurevisor/pull/104 saw `getVariation()` to only return strings, meaning we can avoid all the type-specific methods going forward here.